trunk/ii02823/task_01/src/task_01.cpp

```diff
@@ -0,0 +1,30 @@
+#include <iostream>
+#include <cmath>
+
+using namespace std;
+
+int main()
+{
+    setlocale(LC_ALL, "Ru");
+
+    double y, u, a, b, c, d;
+    cout << "Enter y and u values:\n";
+    cin >> y >> u;
+    cout << "Enter a, b, c, d values:\n";
+    cin >> a >> b >> c >> d;
```

Comment:
It's very nice to declare the variable declarations by groupes.
Also you can split the declarations to match the input blocks more naturally:
```suggestion
    double y, u;
    cout << "Enter y and u values:\n";
    cin >> y >> u;
    
    double a, b, c, d;
    cout << "Enter a, b, c, d values:\n";
    cin >> a >> b >> c >> d;
```